integration_tests/CMakeLists.txt

```
@@ -308,6 +308,7 @@ RUN(NAME bindc_05   LABELS llvm c
         EXTRAFILES bindc_05b.c)
 RUN(NAME bindc_06   LABELS llvm c
         EXTRAFILES bindc_06b.c)
+RUN(NAME test_global_decl    LABELS llvm c)
```

Always test CPython as well:
```suggestion
RUN(NAME test_global_decl    LABELS cpython llvm c)
```